apps/web/playwright/profile.e2e.ts

```diff
@@ -287,6 +285,7 @@ test.describe("Update Profile", () => {
 
     await user.apiLogin();
     await page.goto("/settings/my-account/profile");
+    await page.waitForLoadState("networkidle");
```

Comment:
The page is still hydrating when add secondary email button is clicked, causing hydration error.